lib/Ohmbrewer_Heating_Element.cpp

```diff
@@ -50,6 +104,7 @@ Ohmbrewer::HeatingElement::~HeatingElement() {
  */
 int Ohmbrewer::HeatingElement::doWork() {
     // TODO: Implement HeatingElement::doWork
+    //digitalWrite( someWayOfGettingTheSinglePin(), (getState() : HIGH ? LOW) )
```

Comment:
Remove this commented-out stuff above or you're probably going to have a bad time merging, etc.